### ISSUE TITLE
User failed to login on supabase

### ISSUE THREAD

**Issue by louishugens** [supabase-community/firebase-to-supabase#7]
Hello guys,

First of all, thanks a lot for this really helpful guide. Well, i manage to migrate the users to supabase but they failed to authenticate. How do i use this guide effectively? Thanks

**Comment by burggraf** [comment]
Do you get an error message or can you provide any more information on the error you're getting?  Have you tried resetting the user's password?

**Comment by matthiasmyreha** [comment]
Having the same issue. After I imported the users into supabase and tried to login with one using the firebase email/password combination I'm getting an "invalid login credentials" error. 

I saw that the password is never set in createUser() in import_users.ts (see [this ](https://github.com/supabase-community/firebase-to-supabase/blob/a22c04d2e05beb5348bded5844136229847ec1ff/auth/import_users.ts#L253) line). Can you extend the guide on how and when the passwords are copied? 

Thank you

**Comment by oper2k** [comment]
Same issue here

**Comment by f-bog** [comment]
Yeah would really like to get some input on this. Failing authentication. I am adding the encrypted passwords in the line mentioned by @matthiasmyreha, but still no luck.

**Comment by idrissdjio** [comment]
I am facing the same issue. After debugging, I noticed that the user firebaseAuth data stores the passwords in Hash encryption while Supabase only supports bcrypt. I tried to convert the passwords from JSON into bcrypt format using a couple of alternative but still didn't work.

**Comment by dabrady** [comment]
Has anyone figured out the missing steps here? Are we stuck with somehow forcing users to reset their passwords the next time they open our apps post-migration?
